finat/point_set.py

```diff
@@ -68,6 +68,43 @@ def expression(self):
         return gem.Literal(self.point)
 
 
+class UnknownPointSingleton(AbstractPointSet):
+    """A point set representing a single point with unknown location but known
+    `gem.Variable` expression.
+
+    Stored points are `numpy.nan`s to indicate that they are not known.
+
+    In the future this can be easily replaced with an equivalent for a whole
+    point set representing multiple points though storing the full array of
+    `numpy.nans` may become undesirable and require interface changes."""
```

Comment:
The comments I made on the old PR still apply here.

Reply:
Note old comment
> This is a comment, not a docstring.